### DISCUSSION TITLE
Need assistance with installation getting stuck

### DISCUSSION THREAD

**Discussion by shNNza** [snipe/snipe-it#15643] [General]
Hi there all, please can you point me in the correct direction. I'm loading Snipe IT on a Ubuntu Server via Azure. I keep getting stuck at the point of the in flight setup with error 500. I've put it into debug mode and not sure how to fix this error.

Many thanks

![image](https://github.com/user-attachments/assets/c6fe785e-2de8-4822-bf04-406dba961085)

**Comment by snipe** [comment]
https://snipe-it.readme.io/docs/debugging-permissions

**Comment by shNNza** [comment]
Still a bit lost here. Am giving permissions to the docker folder as the per this ( sudo chown -R my-user:www-data /var/www/snipe-it). /var/www/snipe-it does not exists.

All i have in the docker folder is this below.

![image](https://github.com/user-attachments/assets/746c83c1-276e-4219-a57b-b4e9c26426df)

![image](https://github.com/user-attachments/assets/42a0d972-8108-40e2-a292-9cf1bd37c8e8)

**Reply by simisola88** [reply]
Can you confirm the version of ubuntu you are using
Are you using the docker installation?
or are running it on the host?

I followed the steps here 
https://snipe-it.readme.io/docs/installation and was able to deploy on a shared hosting Cpanel account

**Reply by RevMichaelHutchinson** [reply]
Would you share where the sql database is to use to upload if I use the PUBLIC folder as the website files. I have a cpanel shared account and want to upload Snipe-IT in there as a website for me to use.
